### PR TITLE
fix(themetoggle): support proper dark/light mode, with or without JS

### LINKED ISSUE #382: [FEAT] Dark mode without JS
**Describe the bug**
Without Javascript, the page is 
ouch my eyes
system color scheme is dark
i expect the page to be dark

**To Reproduce**

1. Firefox
2. Disable Javascript
3. go to https://hearch.co/
4. Open developer tools, Inspector, Rules, click the "moon" icon

**Expected behavior**
The page should turn dark.

**Screenshots**
image

**What's your setup? (please complete the following information):**
https://hearch.co/

**Additional context**
I think the default should be to use system color scheme with CSS.

At the very least, put this at the top of the site's CSS file:

```
:root {
  color-scheme: light dark;
}
```

### PR DESCRIPTION
**Pull request by iacore**
Closes #382 

I have no idea what I'm doing but it works

https://tailwindcss.com/docs/dark-mode

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
 /** @type {import('tailwindcss').Config} */
 export default {
-	darkMode: 'selector',
```
Comment (aleksasiriski) [inline tailwind.config.js:3]: CRAAZY!! Amazing IA, I love this! I will test and review ASAP.

Comment (iacore) [reply tailwind.config.js:3]: my name is perhaps, most likely not IA

**Review by aleksasiriski — CHANGES_REQUESTED** [review]
LGTM, just run `make format` and `make lint` and commit afterwards if all is good. I found some formatting problems with TW config JS file.

File: src/lib/components/gadgets/timer/main.svelte [hunk 2]
```diff
@@ -15,8 +15,6 @@
 
 	let beeping = $state(false);
 	$effect(() => {
```
Comment (aleksasiriski) [inline src/lib/components/gadgets/timer/main.svelte:17]: Well that's not desirable, did the formatter remove this or did you do it?

File: src/lib/components/gadgets/timer/main.svelte [hunk 3]
```diff
@@ -15,8 +15,6 @@
...
-		if (query) {
```
Comment (aleksasiriski) [inline src/lib/components/gadgets/timer/main.svelte:18]: This needs to exist, but I see it's creating a linting error, to fix that set it to this:
```js
$effect(() => {
    query; // Re-run effect when query changes.
    beeping = false;
});
```

**Review by aleksasiriski — CHANGES_REQUESTED** [review]
Forgot about this linting problem.

**Review by aleksasiriski — APPROVED** [review]
LGTM, just needs to be squashed merged

File: src/lib/components/gadgets/timer/main.svelte (hunk 3 above)
Comment (iacore) [reply src/lib/components/gadgets/timer/main.svelte]: i forgot svelte reactivity is static-analysis-based...

**Pull request merged**